src/test/kotlin/net/atos/client/sd/model/SdFixtures.kt

```diff
@@ -1,7 +1,134 @@
 package net.atos.client.sd.model
 
+import net.atos.client.sd.model.templates.DocumentsStructure
+import net.atos.client.sd.model.templates.GroupsAccess
+import net.atos.client.sd.model.templates.HeadersStructure
+import net.atos.client.sd.model.templates.Template
+import net.atos.client.sd.model.templates.TemplateGroup
+import net.atos.client.sd.model.templates.TemplatesResponse
+import net.atos.client.sd.model.templates.TemplatesStructure
+import net.atos.client.sd.model.templates.User
+import net.atos.client.sd.model.templates.UserGroup
+import net.atos.client.sd.model.templates.UsersStructure
+import net.atos.client.sd.model.wizard.WizardResponse
+import java.util.UUID
+
 fun createWizardResponse(
     ticket: String = "dummyTicket",
 ) = WizardResponse().apply {
     this.ticket = ticket
 }
+
+fun createUserGroup(name: String) = UserGroup().apply {
+    this.groupsAccess = GroupsAccess().apply {
```

Comment:
`this.` is not required in the context of the `apply` code block. I would remove it unless there's a clash with other variables or parameters.